lib/ansible/plugins/lookup/file.py

```diff
@@ -25,6 +26,10 @@
         type: bool
         required: False
         default: False
+      encodings:
+        description: encoding types of files to be read
+        default: ['utf-8', 'utf-16']
```

Comment:
Please do not make this a list, it should be singular.  We have other modules and plugins that have similar functionality and they are singular.  We should try to maintain consistency.  The default should only by `utf-8` and the user must be responsible for explicitly setting it to a singular value.

Reply:
It's been reverted to a single string parameter.

Reply:
@sivel Please review